src/styles/container.scss

```diff
@@ -0,0 +1,9 @@
+.container {
+  display: flex;
+  flex-wrap: wrap;
```

Comment:
The instructions require that you use grid for cards with different number of columns. Your current implementation uses Flexbox, not CSS Grid.